bcipy/signal/evaluate/rules.py

```diff
@@ -0,0 +1,72 @@
+"""rules.py"""
+
+from bcipy.helpers.load import load_json_parameters
+import numpy as np
+
+
+class Rule:
```

Comment:
ABC with defining @abstractmethods: https://docs.python.org/3/library/abc.html

Reply:
This is a cool module! Implemented Rule as an ABC and made "isBroken" an abstractmethod.